.tx/config

```diff
@@ -36,38 +36,255 @@ source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-intervention_types-intervention_types-yml]
+file_filter  = config/locales/<lang>/views/intervention_types/intervention_types.yml
+source_file  = config/locales/views/intervention_types/intervention_types.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-interventions-interventions-yml]
+file_filter  = config/locales/<lang>/views/interventions/interventions.yml
+source_file  = config/locales/views/interventions/interventions.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-jobs-jobs-yml]
+file_filter  = config/locales/<lang>/views/jobs/jobs.yml
+source_file  = config/locales/views/jobs/jobs.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-programme_types-programme_types-yml]
+file_filter  = config/locales/<lang>/views/programme_types/programme_types.yml
+source_file  = config/locales/views/programme_types/programme_types.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-pupils-pupils-yml]
+file_filter  = config/locales/<lang>/views/pupils/pupils.yml
+source_file  = config/locales/views/pupils/pupils.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-school_groups-school_groups-yml]
+file_filter  = config/locales/<lang>/views/school_groups/school_groups.yml
+source_file  = config/locales/views/school_groups/school_groups.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:views-schools-schools-yml]
 file_filter  = config/locales/<lang>/views/schools/schools.yml
 source_file  = config/locales/views/schools/schools.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
 
+[o:energy-sparks:p:energy-sparks:r:views-schools-transport_surveys-yml]
+file_filter  = config/locales/<lang>/views/schools/transport_surveys.yml
+source_file  = config/locales/views/schools/transport_surveys.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-scoreboards-scoreboards-yml]
+file_filter  = config/locales/<lang>/views/scoreboards/scoreboards.yml
+source_file  = config/locales/views/scoreboards/scoreboards.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:views-shared-shared-yml]
 file_filter  = config/locales/<lang>/views/shared/shared.yml
 source_file  = config/locales/views/shared/shared.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
 
+[o:energy-sparks:p:energy-sparks:r:views-terms_and_conditions-terms_and_conditions-yml]
+file_filter  = config/locales/<lang>/views/terms_and_conditions/terms_and_conditions.yml
+source_file  = config/locales/views/terms_and_conditions/terms_and_conditions.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
```

Comment:
are configs needed for `activerecord.yml`, `common.yml`, `default.yml` and `devise.yml`? (maybe not as in the locale dir - but figured best to double check)

Reply:
The config should have an entry for every YAML file, except anything in the `cy` directory. I've checked and these are all present, as expected.